environments/daml-sdk-env/build/1_installprerequisites.sh

```diff
@@ -1,5 +1,8 @@
 #!/bin/bash
 
+# Configuring the environment for more file watchers
+echo "fs.inotify.max_user_watches=524288" >> /etc/sysctl.conf
+
```

Comment:
I think you should do a `sysctl -p` here

Reply:
Would this work better?